### PR TITLE
Fix security issues, refactor REST API (a little bit)

### PR DESCRIPTION
**Pull request by ruslanpalagin**
no task

### CODE REVIEW

File: api/src/features/notification/notification.service.ts [hunk 1]
```diff
@@ -1,36 +1,60 @@
...
+      const data = {
+        channel,
+        text: `Environment: ${this.configService
+          .get('SUB_ENV')
+          .toUpperCase()}. ${text}`,
+      }
+      await this.callSlack(this.url + '/chat.postMessage', data)
       this.logger.debug(`Message send`);
     } catch (e) {
+      console.log(e.toString())
```
Comment (boivlad) [inline api/src/features/notification/notification.service.ts:34]: Maybe [Object Object] in some cases

File: frontend/app/core/http-client/apiPaths.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export const apiPaths = {
+  createAsset() {
+    return '/assets'
+  },
+  updateAsset() {
+    return '/assets'
+  }
```
Comment (boivlad) [inline frontend/app/core/http-client/apiPaths.ts:7]: Mabe should use parameters, not functions for example:
{
createAsset: '/assets',
updateAsset:  '/assets',
}

File: frontend/app/core/http-client/config.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export const baseUrl = {
+  'localhost': 'http://localhost:3001',
+  'localhost.com': 'http://localhost:3001',
+  'staging.binaryx.com': 'https://api-staging.binaryx.com',
+  'dev.binaryx.com': 'https://api-dev.binaryx.com',
+  'binaryx.com': 'https://api.binaryx.com',
+  'app.binaryx.com': 'https://api.binaryx.com',
+  'blue.binaryx.com': 'https://api-blue.binaryx.com',
+  'green.binaryx.com': 'https://api-green.binaryx.com',
+}[typeof window !== 'undefined' ? window.location.hostname : ''] || ''
```
Comment (boivlad) [inline frontend/app/core/http-client/config.ts:10]: Should add an FF flag for switch server URLs, like configs?

**Review by boivlad — APPROVED** [review]

File: frontend/app/core/http-client/config.ts (hunk 3 above)
Comment (ruslanpalagin) [reply frontend/app/core/http-client/config.ts:10]: It's there 
getFlag('FF_API_URL') || baseUrl

File: frontend/app/core/http-client/apiPaths.ts (hunk 2 above)
Comment (ruslanpalagin) [reply frontend/app/core/http-client/apiPaths.ts:7]: well yes, but some urls has to has an id as an argument or something else.
So function to beconsistent

File: api/src/features/notification/notification.service.ts (hunk 1 above)
Comment (ruslanpalagin) [reply api/src/features/notification/notification.service.ts:34]: yes - plan to rework logs

**Comment by ruslanpalagin** [comment]
Thanks!

**Pull request merged**